app/components/candidate_interface/new_reference_status_line_component.rb

```diff
@@ -0,0 +1,17 @@
+module CandidateInterface
+  class NewReferenceStatusLineComponent < NewReferenceHistoryComponent
+    attr_reader :reference
+
+    def request_sent_at
+      reference.requested_at
+    end
+
+    def can_be_cancelled
+      history.find { |event| event.name == 'request_sent' }
+    end
```

Comment:
It might not make a difference to how we’re calling this, but I think this logic could be simplified to only referenced in the "requested" state? (Otherwise it’d return true even if the reference had already been cancelled).

This logic is also used on the view itself (see [view_reference.html.erb#L45](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/views/candidate_interface/offer_dashboard/view_reference.html.erb#L45)) so maybe worth extracting into the `ReferenceActionsPolicy` class alongside `can_send_reminder?`?

```suggestion
    def can_be_cancelled
      reference.feedback_requested?
    end
```